openassessment/templates/openassessmentblock/staff/oa_staff_cancelled.html

```diff
@@ -0,0 +1,22 @@
+{% load i18n %}
+
+<li id="openassessment__staff-assessment" class="openassessment__steps__step step--staff-assessment">
```

Comment:
The id is part of the expected structure for each of the UI_MODELS, where a placeholder element is shown during loading time (see https://github.com/edx/edx-ora2/blob/master/openassessment/xblock/openassessmentblock.py#L44). I can't use a class instead of an ID without changing the general expected structure of steps in the LMS view, and I think that is beyond the scope of this story.